src/Passport.php

```diff
@@ -44,9 +45,16 @@ class Passport
      * @var array
      */
     public static $scopes = [
-        //
+		//
```

Comment:
Scopes variable with all scopes is needed for backport but also for Oauth2 to work